app/permissions/PermissionsActionCheck.scala

```diff
@@ -85,6 +88,18 @@ class EditEditorialFrontsPermissionCheck(val acl: Acl)(implicit ec: ExecutionCon
   override val restrictedAction: String = "Edit editorial fronts."
 }
 
+class EditEmailFrontsPermissionCheck(val acl: Acl)(implicit ec: ExecutionContext) extends PermissionActionFilter {
+  override implicit val executionContext: ExecutionContext = ec
+  override val testAccess: String => Authorization = (email: String) => {
+    val emailAccess = acl.testUser(Permissions.LaunchAndEditEmailFronts, "facia-tool-allow-edit-email-fronts-for-all")(email)
+    emailAccess match {
+      case AccessGranted => emailAccess
+      case AccessDenied => new EditEditorialFrontsPermissionCheck(acl)(executionContext).testAccess(email)
+    }
+  }
+  override val restrictedAction: String = "Edit email fronts."
+}
```

Comment:
nit: I think if a user has neither permission, this will show a message saying unauthorized to edit editorial fronts, rather than unauthorized to edit email fronts. It might be nice to warn/suggest the narrower permission first?

Reply:
We'd definitely want to avoid that – is there a side effect in this method when we delegate to `EditEditorialFrontsPermissionCheck`? The return value should be just `AccessDenied` or `AccessGranted`, so I'm unsure how the caller would distinguish between the two paths – unless that function has a side-effect I've missed 🤔 

In general, I don't think users see a particular permission when access is denied in Fronts, although it may appear in logs:
<img width="622" alt="Screenshot 2021-11-26 at 11 59 29" src="https://user-images.githubusercontent.com/7767575/143577794-67a0fbca-e9fc-488f-b25b-2b0d189c4314.png">

Reply:
You're completely right! I've misunderstood how the restrictedAction was being used, ignore this comment :)

Reply:
Ah, gotcha, no problem!